runtime/int.go

```diff
@@ -430,6 +441,20 @@ func intCheckedMul(v, w int) (int, bool) {
 	return x, true
 }
 
+func intCheckedPow(v, w int) (int, bool) {
+	// first try the shortcut - assuming the this operation is
```

Comment:
Yes. And the implementation of python x ** y differs a lot from Go big.Exp and math.Pow when x or y are negative.

A complete implementation will need some more time ...